roadmap/implementers-guide/src/types/overseer-protocol.md

```diff
@@ -126,20 +126,60 @@ enum CandidateSelectionMessage {
 }
 ```
 
+## Collator Protocol Message
+
+Messages received by the [Collator Protocol subsystem](../node/collators/collator-protocol.md)
+
+```rust
+enum CollatorProtocolMessage {
```

Comment:
"Protocol" suggests that this message is passed to other nodes, and is somewhat stable, but as I understand this is just an internal message between subsystems and could change whenever. How about `CollatorInfoMessage` or just `CollatorInfo`?

Reply:
The subsystem is called the Collator Protocol subsystem. Our naming convention for these message types is `format!("{}Message", subsystem_name)`